using/legal/code-of-conduct.md

```diff
@@ -77,6 +77,6 @@ If you say something that is found offensive, and you are called out on it, try
 
 ## History
 
-This policy was initially adopted from the Front-end London Slack community and has been modified since. A version history can be seen on [GitHub](https://github.com/exercism/website-copy/edit/main/pages/code_of_conduct.md).
+This policy was initially adopted from the Front-end London Slack community and has been modified since. A version history can be seen on [GitHub](https://github.com/exercism/docs/blob/main/using/legal/code-of-conduct.md).
```

Comment:
Maybe link to the history of the file?

```suggestion
This policy was initially adopted from the Front-end London Slack community and has been modified since. A version history can be seen on [GitHub](https://github.com/exercism/docs/commits/main/using/legal/code-of-conduct.md).
```